cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/runtimejob/DataprocJarUtil.java

```diff
@@ -101,7 +109,7 @@ public boolean accept(String className, URL classUrl, URL classPathUrl) {
     return createLocalFile(location, false);
   }
 
-  private static LocalFile createLocalFile(Location location, boolean archive) throws IOException {
+  static LocalFile createLocalFile(Location location, boolean archive) throws IOException {
```

Comment:
nit: I think it's more understandable if we name this to getLocalFile(), since it's not actually creating anything.

Reply:
ack.